Cargo.toml

```diff
@@ -124,7 +124,7 @@ rolldown_workspace = { version = "0.1.0", path = "./crates/rolldown_workspace" }
 
 anyhow = "1.0.86"
 append-only-vec = { version = "0.1.5" }
-arcstr = "1.2.0"
+arcstr = { version = "1.2.0", features = [] }
```

Comment:
Does this do the trick? Should we default-features = false?

Reply:
Forgot it's a default feature.